### PR TITLE
Test other form of explicit stretching

### PR DESCRIPTION
**Pull request by rwlbuis**
Grid item sizing can be done explicitly using align-items:stretch/justify-items: stretch as long as align-self:auto/justify-self:auto are used on the items, so add tests to verify it has the same result as using align-self:stretch/justify-self:stretch.

### CODE REVIEW

File: css/css-sizing/aspect-ratio/grid-aspect-ratio-018.html [hunk 1]
```diff
@@ -17,6 +17,9 @@
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
 
 <div id="reference-overlapped-red"></div>
-<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 100px;">
-  <div style="height: 100px; aspect-ratio: 1/2; justify-self: stretch; background: green;"></div>
+<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 50px;">
```
Comment (javifernandez) [inline css/css-sizing/aspect-ratio/grid-aspect-ratio-018.html:20]: Why we need to change the grid-template-rows value to 50px ?

File: css/css-sizing/aspect-ratio/grid-aspect-ratio-019.html [hunk 2]
```diff
@@ -17,6 +17,9 @@
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
 
 <div id="reference-overlapped-red"></div>
-<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 100px;">
+<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 50px;">
```
Comment (javifernandez) [inline css/css-sizing/aspect-ratio/grid-aspect-ratio-019.html:20]: Why this change ?

File: css/css-sizing/aspect-ratio/grid-aspect-ratio-018.html (hunk 1 above)
Comment (rwlbuis) [reply css/css-sizing/aspect-ratio/grid-aspect-ratio-018.html:20]: To keep displaying 100x100 green rectangles like before.

File: css/css-sizing/aspect-ratio/grid-aspect-ratio-019.html (hunk 2 above)
Comment (rwlbuis) [reply css/css-sizing/aspect-ratio/grid-aspect-ratio-019.html:20]: Same, to keep displaying 100x100 green rectangles like before.

**Review by javifernandez — APPROVED** [review]
